starknet_py/abi/v0/parser_test.py

```diff
@@ -189,16 +189,6 @@ def test_missing_type_used(missing_name, input_dict):
         AbiParser([input_dict]).parse()
 
 
-def test_deserialize_proxy_abi():
```

Comment:
I agree with you, but there is an identical test below that also `Contains all types of ABI apart from structures`, so because of its redundancy I think It can be removed